default/python/AI/AIDependencies.py

```diff
@@ -668,13 +670,13 @@ class Tags:
 
 # <editor-fold desc="Piloting traits">
 # TODO (Morlic): Consider using only 1 dict with tuple for (Capacity, SecondaryStat) effects
-PILOT_DAMAGE_MODIFIER_DICT = {
+PILOT_DAMAGE_UNSCALED_MODIFIER_DICT = {
     # TRAIT:    {weapon_name: effect, weapon_name2: effect2,...}
     "NO":       {},
     "BAD":      {"SR_WEAPON_1_1": -1, "SR_WEAPON_2_1": -2, "SR_WEAPON_3_1": -3, "SR_WEAPON_4_1": -5},
     "GOOD":     {"SR_WEAPON_1_1":  1, "SR_WEAPON_2_1":  2, "SR_WEAPON_3_1":  3, "SR_WEAPON_4_1": 5},
     "GREAT":    {"SR_WEAPON_1_1":  2, "SR_WEAPON_2_1":  4, "SR_WEAPON_3_1":  6, "SR_WEAPON_4_1": 10},
-    "ULTIMATE": {"SR_WEAPON_1_1":  3, "SR_WEAPON_2_1":  6, "SR_WEAPON_3_1":  9, "SR_WEAPON_4_1": 15, "SR_WEAPON_0_1": 1},
```

Comment:
> Probably it should be 1 in all cases.
> 
> this was introduced in #814

This simulates the species pilot trait effect for flak damage, increasing the damage from 1 to two per shot for ultimate pilot species.
As damage for flak is meaningless since we introduced targeting (flak does only shoot at fighters and fighters do not have structure) I removed the meaningless effect and the corresponding AI entry.

I could set  "SR_WEAPON_0_1": 0 in all maps, but it does not really add much info and i do not have the possibility to playtest right now. So if there are no other gripes I would rather leave it like it is.

Reply:
@Cjkjvfnby do you have any other review comments but setting SR_WEAPON_0_1 to zero in all maps?

Reply:
> I removed the meaningless effect and the corresponding AI entry

I didn't know this. 


Let leave it as is.